package.json

```diff
@@ -39,13 +39,15 @@
 		"ts-node": "^10.9.1",
 		"tslib": "^2.4.1",
 		"typescript": "^4.9.3",
-		"vite": "^4.0.0",
+		"vite": "^4.4.9",
 		"vitest": "^0.25.3"
 	},
 	"type": "module",
 	"dependencies": {
 		"cross-fetch": "^4.0.0",
 		"crypto-js": "^4.1.1",
-		"js-sha256": "^0.9.0"
+		"js-sha256": "^0.9.0",
```

Comment:
Can you remind me why this package is required when you can use something like `Crypto` or `SubtleCrypto`?

Reply:
SubtleCrypto has a `.verify` not sure if that helps.

Reply:
you're right, js-sha256 isn't used anywhere

Reply:
```.verify``` is not needed here because we are the ones creating the codeChallenge and the server is the one validating it.